types/node/node-tests.ts

```diff
@@ -1365,6 +1365,10 @@ namespace dgram_tests {
         });
         ds.send(new Buffer("hello"), 5000, "127.0.0.1");
         ds.setMulticastInterface("127.0.0.1");
+        ds.getRecvBufferSize();
```

Comment:
Tests are also present around 100 lines below :smile:

Reply:
Ok, I close this PR and close myself in the bathroom to cry 😢 😢 😢 😢 😢 😢